### DISCUSSION TITLE
All Books Are tagged as a Series

### DISCUSSION THREAD

**Discussion by markmccoid** [advplyr/audiobookshelf#2897] [Q&A] [answered]
Even though my m4b file does not have the SERIES or SERIES-PART tags, every book when scanned gets added to a series with the Author's name.

Has anyone else seen this?

**Comment by advplyr** [comment, marked as answer]
This is most likely your folder structure

**Reply by markmccoid** [reply]
Could it be because I wanted to separate Fiction and Nonfiction?

So I have 
Fiction -> Author -> Title 
            -> Author -> Title 
Nonfiction -> Author -> Title 
                   -> Author -> Title

**Reply by markmccoid** [reply]
That was the issue.  Still learning about Audiobook Shelf.  Great app@

**Reply by advplyr** [reply]
The website has the supported folder structures https://www.audiobookshelf.org/docs#book-directory-structure
